Sources/_CryptoExtras/RSA/RSA+BlindSigning.swift

```diff
@@ -0,0 +1,414 @@
+    ///
+    /// This named variant uses SHA-384 as the EMSA-PSS Hash option, MGF1 with SHA-384 as the EMSA-PSS MGF option,
+    /// and 0 as the EMSA-PSS sLen option (0-byte salt length); it also uses the randomized preparation function.
+    ///
+    /// - Seealso: [RFC 9474: RSABSSA Variants](https://www.rfc-editor.org/rfc/rfc9474.html#name-rsabssa-variants).
+    public static let RSABSSA_SHA384_PSSZERO_Randomized = Self<SHA384>(padding: .PSSZERO, preparation: .randomized)
+
+    /// RSABSSA-SHA384-PSS-Deterministic
+    ///
+    /// This named variant uses SHA-384 as the EMSA-PSS Hash option, MGF1 with SHA-384 as the EMSA-PSS MGF option,
+    /// and 48 as the EMSA-PSS sLen option (48-byte salt length); it also uses the identity preparation function.
+    ///
+    /// - WARNING: Not all named variants can be used interchangeably. In particular, applications that provide
+    /// high-entropy input messages can safely use named variants without randomized message preparation, as the
+    /// additional message randomization does not offer security advantages.
+    /// For all other applications, the variants that use the randomized preparation function protect clients from
+    /// malicious signers.
+    ///
+    /// - Seealso: [RFC 9474: RSABSSA Variants](https://www.rfc-editor.org/rfc/rfc9474.html#name-rsabssa-variants).
+    public static let RSABSSA_SHA384_PSS_Deterministic = Self<SHA384>(padding: .PSS, preparation: .identity)
+
+    /// RSABSSA-SHA384-PSSZERO-Deterministic
+    ///
+    /// This named variant uses SHA-384 as the EMSA-PSS Hash option, MGF1 with SHA-384 as the EMSA-PSS MGF option,
+    /// and 0 as the EMSA-PSS sLen option (0-byte salt length); it also uses the identity preparation function
+    ///
+    /// - NOTE: This is the only variant that produces deterministic signatures over the client's input message.
+    ///
+    /// - WARNING: Applications that require deterministic signatures can use the RSABSSA-SHA384-PSSZERO-Deterministic
+    /// variant, but only if their input messages have high entropy. Applications that use
+    /// RSABSSA-SHA384-PSSZERO-Deterministic SHOULD carefully analyze the security implications, taking into account
+    /// the possibility of adversarially generated signer keys as described in Section 7.3. When it is not clear whether
+    /// an application requires deterministic or randomized signatures, applications SHOULD use one of the variants with
+    /// randomized message preparation.
+    ///
+    /// - Seealso: [RFC 9474: RSABSSA Variants](https://www.rfc-editor.org/rfc/rfc9474.html#name-rsabssa-variants).
+    public static let RSABSSA_SHA384_PSSZERO_Deterministic = Self<SHA384>(padding: .PSSZERO, preparation: .identity)
+}
+
+extension _RSA.BlindSigning {
+    /// An input ready to be blinded, possibly prepended with random bytes.
+    ///
+    /// Users should not attempt to create values of this type; it is created and returned by the prepare operation.
```

Comment:
There's no should not here, they cannot.

Reply:
Heh, I meant "should not attempt" because "cannot" but I've streamlined the language to just "cannot" here and elsewhere.